.github/workflows/build.yml

```diff
@@ -54,6 +66,6 @@ jobs:
 
     - name: Script
       run: |
+        pip --verbose list
         python3 CI/check-nnpy.py
         ./CI/run_tests.sh
-        sudo python3 /usr/local/bin/nose2 utests.tests.test
```

Comment:
FYI for reviewers - these tests are still run, but I moved the command that runs them into the `./CI/run_tests.sh` script, for easier control of additional command line options.